xtevent/xtevent.sthlp

```diff
@@ -115,19 +115,19 @@ and {opt overidpost()} (See below).
 {phang2} {opt pre} is the number of pre-event periods where anticipation effects are allowed. With {opt window}, {opt pre} is 0.
 
 {phang2} {opt post} is the number of post-event periods where policy effects are allowed. With {opt window}, {opt post} is the number
-of periods after the event minus 2.
+of periods after the event (not including the period for the event, e.g. event time = 0).

```

Comment:
In -xtevent- when translating values from -post- and -overidpost- into the righ-hand side value of -window-, post is the number of post-event coefficients ( i.e., not including the event-time coefficient) except the lastest two coefficients, assigned to overidpost for the leveling off test.
https://github.com/JMSLab/xtevent/blob/d2111190adaa71e6af655c0d2ceb1dbe722a2631/xtevent/xtevent.ado#L184